.github/workflows/m68k.yml

```
@@ -88,6 +91,19 @@ jobs:
         sudo mount debian-m68k.img vm
         sudo cp $(which qemu-m68k-static) vm/usr/bin/
 
+    - name: Prepare for build with target as JSON spec
+      run: |
+        ./y.sh prepare --only-libcore --cross
+
+    - name: Build sysroot with target as JSON spec
+      run: |
+        ./y.sh build --build-sysroot --target-triple m68k-unknown-linux-gnu --target ${{ github.workspace }}/target_specs/m68k-unknown-linux-gnu.json
+
+    - name: Build with target as JSON spec
+      run: |
+        cargo build --target ${{ github.workspace }}/target_specs/m68k-unknown-linux-gnu.json
```

This will require the use of `y.sh`:
```suggestion
        ./y.sh cargo build --target ${{ github.workspace }}/target_specs/m68k-unknown-linux-gnu.json
```

---

This also requires setting `OVERWRITE_TARGET_TRIPLE` for now. I edited my post above.

---

OVEWRITE_TARGET_TRIPLE is defined at the top as env variable